solutions/swb-app/src/schemas/environmentDatasetMetadataParser.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import { z } from 'zod';
+
+// eslint-disable-next-line @rushstack/typedef-var
+export const EnvironmentDatasetMetadataParser = z
```

Comment:
Nitpick: I would name this `DatasetEnvironmentMetadataParser` since DS is the pk and environment is the sk